app/docs/dev/policies/meshtrace.md

```diff
@@ -42,17 +42,96 @@ For HTTP you can also manually forward the following headers:
 | MeshGatewayRoute  | ✅        | ❌  | ❌   |
 | MeshHTTPRoute     | ❌        | ❌  | ❌   |
 
-If you don't understand this table you should read [matching docs](matching#policy-matching)
+If you don't understand this table you should read [matching docs](../matching#policy-matching).
```

Comment:
Isn't it in the same directory?

Reply:
That's how these docs are like now. meshtrace is a path in the URL